GoogleCloudPlatform/hpc-toolkit
# Update lustre to latest

This PR updates the DDN-Exascaler commit reference to the latest version released. 

This version change includes a few new disk types.
- hyperdisk-balanced
- hyperdisk-extreme 
- hyperdisk-throughput

Along with the new disk types, there were two new parameters.
- disk_iops
- disk_mbps

**Manual testing**
To test the new disks and parameters, I used `examples/pfs-lustre.yaml` and changed the defaults for lustre. Below is an example of testing to ensure `hyperdisk-throughput` was working as intended. After deploying, I verified it was successful on Pantheon. 
```
  - id: lustre
    source: community/modules/file-system/DDN-EXAScaler
    use: [network1]
    settings:
      local_mount: /lustre
      oss:
        node_type: "n2-standard-16"
        node_cpu: "Intel Cascade Lake"
        nic_type: "GVNIC"
        public_ip: true
        node_count: 1
      ost: 
        disk_bus: "SCSI"
        disk_type: "hyperdisk-throughput"
        disk_iops: null
        disk_mbps: 20
        disk_size: 2048
        disk_count: 1
        disk_raid: false
```
All the available disk types were tested in a similar fashion.
